### ISSUE TITLE
build failure in @nodert-win10-rs4/windows.devices.bluetooth.genericattributeprofile

### ISSUE THREAD

**Issue by WimYedema** [NodeRT/NodeRT#116]
Hi,

I get a build failure when trying to build this. The error message:

`c:\users\yedema21\git\noble-uwp\node_modules\@nodert-win10-rs4\windows.devices.bluetooth.genericattributeprofile\_nodert_generated.cpp(12045): error C2440: '=': cannot convert from 'Windows::Foundation::IAsyncOperation<Windows::Devices::Bluetooth::GenericAttributeProfile::GattClientNotificationResult ^> ^' to 'Windows::Foundation::IAsyncOperation<Windows::Foundation::Collections::IVectorView<Windows::Devices::Bluetooth::GenericAttributeProfile::GattClientNotificationResult ^> ^> ^' [C:\Users\yedema21\git\noble-uwp\node_modules\@nodert-win10-rs4\windows.devices.bluetooth.genericattributeprofile\build\binding.vcxproj]`

**Comment by WimYedema** [comment]
No, it was a bit to much effort for me...

**Comment by ns16** [comment]
I get the same error with the same namespace.

**Comment by jeremywho** [comment]
@ns16 Yeah, its a bummer. I tried some hacks to get things working for a couple days with no luck. 
Unfortunately, it appears this project is no longer active.

**Comment by zhfnjust** [comment]
I get the same error with the same namespace.

**Comment by zhfnjust** [comment]
@jeremywho @ns16 @WimYedema 
anyone got a solution?

**Comment by nevim42** [comment]
I have the exactly same error. Does have somebody some solution?

**Comment by acstacey** [comment]
It seems that NodeRT has a bug with method overloads having different return types.
@jasongin patched it manually in his noble-uwp project and I added the patch to a nodert-win10-cu generated project that I am using in a noble-uwp fork.

https://github.com/acstacey/nodert-win10-cu-windows.devices.bluetooth.genericattributeprofile
https://github.com/acstacey/noble-uwp/tree/nodert
